scripts/terraform-docs.sh

```diff
@@ -0,0 +1,14 @@
+
+if which terraform-docs >/dev/null; then
```

Comment:
Nice addition! I am going to ask my team to deploy this script in all our Terraform modules and also, add a new Makefile directive to run it.

<img src="https://media4.giphy.com/media/l2JIdnF6aJnAqzDgY/giphy.gif"/>